src/renderer/components/AccountView.tsx

```diff
@@ -52,6 +55,66 @@ function AccountView(props: { pubKey: string | undefined }) {
     undefined
   );
 
+  // create dummy keypair wallet if none is selected by user
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const wallet = useAnchorWallet() || {
+    signAllTransactions: async (
+      transactions: sol.Transaction[]
+    ): Promise<sol.Transaction[]> => Promise.resolve(transactions),
+    signTransaction: async (
+      transaction: sol.Transaction
+    ): Promise<sol.Transaction> => Promise.resolve(transaction),
+    publicKey: new sol.Keypair().publicKey,
```

Comment:
don't do this - there's a createNewKeypair that then also saves the keypair into the config file, so it can be used again later - otherwise, this account will not survive restarting workbench... (which is also why I take everything through the wallet-adapter keypair - its the only safe way.

Reply:
Since it's just a non functional wallet to make the Anchor context happy, I figured a throwaway key would be fine. It won't sign any transactions correctly, but if the user has not selected a wallet in the Workbench yet, what else did you have in mind to do?